networking/__tests__/telemetry.spec.js

```diff
@@ -1,6 +1,6 @@
 import defaults from '../telemetry';
 
-describe('telemetry', () => {
+describe('basic telemetry', () => {
   it('should have name', () => {
```

Comment:
Shouldn't it check for a version here as well?

Reply:
You mean the test case that is below this one? This telemetry file defines name and version for this sdk, and does not include anything else. This is used from the telemetry value generator to obtain these details.